### PR TITLE
[Balance] [QoL] Allow Pirouette Meloetta to be selectable on starter screen

### PR DESCRIPTION
**Pull request by Blitz425**
## What are the changes?
Allows Meloetta's Pirouette Form to be selectable on the starter select.

## Why am I doing these changes?
Purely QoL change, Meloetta is not able to use Pirouette to its fullest extent due to the nature of how PokeRogue plays so this allows player further options if they would want to use this Pokemon. As well as setting it free from its previous restriction of wasting a moveslot to use the form.

## What did change?
Pirouette as a selectable form
Meloetta no longer changes forms between biome changes or trainer battles.

### Screenshots/Videos
image

## How to test the changes?
Pick Meloetta on the starter select
## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [x] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by frutescens** [comment]
Hi, did you let the localization server know about this PR yet?

**Review by frutescens — CHANGES_REQUESTED** [review]
I think some localizations can already be done independently using Bulbapedia. https://bulbapedia.bulbagarden.net/wiki/Meloetta_(Pok%C3%A9mon)#In_other_languages

File: src/locales/de/pokemon-form.ts [hunk 1]
```diff
@@ -89,6 +89,7 @@ export const pokemonForm: SimpleTranslationEntries = {
   "landorusIncarnate": "Inkarnationsform",
   "keldeoOrdinary": "Standardform",
   "meloettaAria": "Gesangsform",
+  "meloettaPirouette": "Pirouette",
```
Comment (frutescens) [inline src/locales/de/pokemon-form.ts:92]:
```suggestion
  "meloettaPirouette": "Tanzform",
```

File: src/locales/es/pokemon-form.ts [hunk 2]
```diff
@@ -88,6 +88,7 @@ export const pokemonForm: SimpleTranslationEntries = {
   "landorusIncarnate": "Incarnate",
   "keldeoOrdinary": "Ordinary",
   "meloettaAria": "Aria",
+  "meloettaPirouette": "Pirouette",
```
Comment (frutescens) [inline src/locales/es/pokemon-form.ts:91]:
```suggestion
  "meloettaPirouette": "Forma Danza",
```

File: src/locales/fr/pokemon-form.ts [hunk 3]
```diff
@@ -88,6 +88,7 @@ export const pokemonForm: SimpleTranslationEntries = {
   "landorusIncarnate": "Avatar",
   "keldeoOrdinary": "Normal",
   "meloettaAria": "Chant",
+  "meloettaPirouette": "Pirouette",
```
Comment (frutescens) [inline src/locales/fr/pokemon-form.ts:91]:
```suggestion
  "meloettaPirouette": "Forme Danse",
```

File: src/locales/it/pokemon-form.ts [hunk 4]
```diff
@@ -88,6 +88,7 @@ export const pokemonForm: SimpleTranslationEntries = {
   "landorusIncarnate": "Incarnazione",
   "keldeoOrdinary": "Normale",
   "meloettaAria": "Canto",
+  "meloettaPirouette": "Pirouette",
```
Comment (frutescens) [inline src/locales/it/pokemon-form.ts:91]:
```suggestion
  "meloettaPirouette": "Forma Danza",
```

File: src/locales/ja/pokemon-form.ts [hunk 5]
```diff
@@ -88,6 +88,7 @@ export const pokemonForm: SimpleTranslationEntries = {
   "landorusIncarnate": "Incarnate",
   "keldeoOrdinary": "Ordinary",
   "meloettaAria": "Aria",
+  "meloettaPirouette": "Pirouette",
```
Comment (frutescens) [inline src/locales/ja/pokemon-form.ts:91]:
```suggestion
  "meloettaPirouette": "ステップフォルム",
```

File: src/locales/ko/pokemon-form.ts [hunk 6]
```diff
@@ -88,6 +88,7 @@ export const pokemonForm: SimpleTranslationEntries = {
   "landorusIncarnate": "화신폼",
   "keldeoOrdinary": "평상시 모습",
   "meloettaAria": "보이스폼",
+  "meloettaPirouette": "Pirouette",
```
Comment (frutescens) [inline src/locales/ko/pokemon-form.ts:91]:
```suggestion
  "meloettaPirouette": "스텝폼",
```

File: src/locales/fr/pokemon-form.ts (hunk 3 above)
Comment (Adri1) [inline src/locales/fr/pokemon-form.ts:91]:
```suggestion
  "meloettaPirouette": "Danse",
```

File: src/locales/pt_BR/pokemon-form.ts [hunk 7]
```diff
@@ -89,6 +89,7 @@ export const pokemonForm: SimpleTranslationEntries = {
   "landorusIncarnate": "Incarnate",
   "keldeoOrdinary": "Ordinary",
   "meloettaAria": "Aria",
+  "meloettaPirouette": "Pirouette",
```
Comment (frutescens) [inline src/locales/pt_BR/pokemon-form.ts:92]:
```suggestion
  "meloettaPirouette": "Forma Pirueta",
```

File: src/locales/zh_CN/pokemon-form.ts [hunk 8]
```diff
@@ -88,6 +88,7 @@ export const pokemonForm: SimpleTranslationEntries = {
   "landorusIncarnate": "化身",
   "keldeoOrdinary": "通常",
   "meloettaAria": "歌声",
+  "meloettaPirouette": "Pirouette",
```
Comment (frutescens) [inline src/locales/zh_CN/pokemon-form.ts:91]:
```suggestion
  "meloettaPirouette": "舞步形态",
```

File: src/locales/zh_TW/pokemon-form.ts [hunk 9]
```diff
@@ -88,6 +88,7 @@ export const pokemonForm: SimpleTranslationEntries = {
   "landorusIncarnate": "Incarnate",
   "keldeoOrdinary": "Ordinary",
   "meloettaAria": "Aria",
+  "meloettaPirouette": "Pirouette",
```
Comment (frutescens) [inline src/locales/zh_TW/pokemon-form.ts:91]:
```suggestion
  "meloettaPirouette": "舞步形態",
```

File: src/locales/fr/pokemon-form.ts (hunk 3 above)
Comment (frutescens) [reply src/locales/fr/pokemon-form.ts:91]: Hi, the official term is 'Forme Danse' but I don't know anything about French so I wanted to get your thoughts first.

**Comment by Adri1** [comment]
@frutescens Please remove "Forme" / "Forma" from French, Spanish and Italian 🙏

File: src/locales/fr/pokemon-form.ts (hunk 3 above)
Comment (Adri1) [reply src/locales/fr/pokemon-form.ts:91]: I know, but shorter versions are used for Pokémon form names when possible, even in English

Comment (frutescens) [reply src/locales/fr/pokemon-form.ts]: That makes sense. Thank you for explaining it to me.

**Comment by Blitz425** [comment]
Not touching jp, zh_cn, zh_tw, as most do not have other forms or Meloetta itself translated, and I cannot accurately tell where the "Pirouette" in the name is.

**Review by CodeTappert — APPROVED** [review]

**Review by damocleas — APPROVED** [review]

**Comment by Adri1** [comment]
> Not touching jp, zh_cn, zh_tw, as most do not have other forms or Meloetta itself translated, and I cannot accurately tell where the "Pirouette" in the name is.

In Japanese, the "Form" part it the four last characters, フォルム

**Review by Adri1 — APPROVED** [review]

**Pull request merged**